### PR TITLE
SUS-2976: remove old image review

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-2976

Corresponds to: https://github.com/Wikia/config/pull/2300

@rybmat @TK-999 @macbre — guys, please check if I did not remove too much (see what I left there) :)

### CODE REVIEW

File: includes/wikia/tasks/Tasks/ImageReviewTask.class.php [hunk 1]
```diff
@@ -111,7 +111,6 @@ public function delete( $pageList, $suppress = false ) {
 
 		$success = $articlesDeleted == count( $pageList );
 		if ( !$success ) {
```
Comment (WikiaTech) [inline includes/wikia/tasks/Tasks/ImageReviewTask.class.php]: MAJOR Either remove or fill this block of code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS108)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: maintenance/wikia/sql/dataware-schema.sql [hunk 2]
```diff
@@ -180,62 +180,6 @@ CREATE TABLE `ignored_users` (
...
-
-DROP TABLE IF EXISTS `image_review_wikis`;
-CREATE TABLE `image_review_wikis` (
-  `wiki_id` int(11) NOT NULL,
-  `top200` tinyint(1) DEFAULT NULL,
-  PRIMARY KEY (`wiki_id`),
-  KEY `top200` (`top200`)
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
-
---
```
Comment (macbre) [inline maintenance/wikia/sql/dataware-schema.sql:238]: Do we have a ticket to remove these tables from the database?

**Review by macbre — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

File: maintenance/wikia/sql/dataware-schema.sql (hunk 2 above)
Comment (ghost) [reply maintenance/wikia/sql/dataware-schema.sql:238]: Yes, it's https://wikia-inc.atlassian.net/browse/SUS-3061

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**